Formula/aria2.rb

```diff
@@ -19,15 +19,12 @@ class Aria2 < Formula
   depends_on "pkg-config" => :build
   depends_on "gettext"
   depends_on "libssh2"
+  depends_on "openssl@3"
```

Comment:
There is linkage on macOS too (via `libssh2`) so may as well declare this explicitly.